bucket/gimp.json

```diff
@@ -65,7 +69,17 @@
     ],
     "checkver": {
         "url": "https://www.gimp.org/downloads/",
-        "regex": "gimp-([\\d.]+)-setup(-\\d)?\\.exe",
-        "replace": "${1}${2}"
+        "regex": "gimp-(?<major>\\d+).(?<minor>\\d+).(?<patch>\\d+)-setup(?<build>-\\d)?\\.exe",
```

Comment:
This would work instead as well

        "regex": "gimp-(?<version>[\\d.]+)-setup(?<build>-\\d)?\\.exe",
        "replace": "${version}${build}"